docker-for-aws/release-notes.md

```diff
@@ -6,41 +6,89 @@ title: Docker for AWS Release Notes
 
 {% include d4a_buttons.md %}
 
-## 1.13.0-1
-Release date: 1/18/2017
+## Stable Channel
+
+### 1.13.1-2 (stable)
+Release date: 02/08/2017
 
 {{aws_button_latest}}
 
-### New
+**New**
```

Comment:
That is fine, we would want to link directly to a release version anyway. It also cleans up the side menu since we don't have New after every release version. Our release notes aren't that big, so it won't be that much of a hassle to our users.